doc-content/enterprise-only/cases/case-management-creating-manager-approval-proc.adoc

```diff
@@ -0,0 +1,89 @@
+* *Groups* - `supplier`
+* *Name* - `Prepare hardware spec`
+* *Task Name* - `PrepareHardwareSpec`
+* *Description* - `Prepare hardware specification for #{initiator} (order number #{CaseId})`
+
+. Click the down arrow in the *Assignments* field, add the following new conditions in the *Prepare hardware spec Data I/O* window. For the first output assignment, select *Custom* for the *Data Type* and input `org.jbpm.document.Document`. Click *Save*.
++
+image::cases/hardware-spec-data.png[Prepare hardware I/O values]
+
+. Click the *Prepare hardware spec* user task and create an new user task.
++
+image::cases/new-user-task.png[Creating a new user task]
+
+. Click the new user task, click image:cases/left-arrows.png[] to open the *Properties* panel, and input the following values:
+* *Actors* - `manager`
+* *Name* - `Manager approval`
+* *Task Name* - `ManagerApproval`
+* *Description* - `Approval request for new hardware for #{initiator} (order number #{CaseId})`
+
+. Click the down arrow in the *Assignments* field, add the following new conditions in the *ManagerApproval Data I/O* window. For the first output assignment, select *Custom* for the *Data Type* and input `org.jbpm.document.Document`. Click *Save*.
++
+image::cases/manager-approval-data.png[Manager approval I/O values]
+
+. Click the *Manager approval* user task and create a Data-based Exclusive (XOR) gateway.
++
+image::cases/xor-gateway.png[Creating an exclusive gateway]
+
+. Click image:cases/right-arrow.png[] to open the *Object Library*, expand *Subprocesses*, click *Reusable*, and drag the new element to the canvas on the right side of the Data-based Exclusive (XOR) gateway.
+. Connect the Data-based Exclusive (XOR) gateway to the subprocess.
++
+image::cases/connect-sub.png[Creating exclusive gateway to subprocess]
+
+.  Click the subprocess, click image:cases/left-arrows.png[] to open the *Properties* panel, and input or select the following values:
+* *ActivityType* - `Sub-Process`
+* *Name* - `Place order`
+* *Called Element* - Select *place-order.bpmn* from the *Asset Name* column, and click *Save*.
+* *Independent* - *false*
+. Click the down arrow in the *Assignments* field, add the following new conditions in the *Place order Data I/O* window, and click *Save*.
++
+image::cases/place-order-io.png[Place order I/O values]
+
+. Click on the connection from the Data-based Exclusive (XOR) gateway to the subprocess and click image:cases/left-arrows.png[] to open the *Properties* panel.
++
+image::cases/connect-sub-order.png[Exclusive gateway to subprocess]
+. Click the down arrow in the *Expression* field and add the following *Sequence Flow Conditions* and click *Ok*.
++
+image::cases/seq-flow-conds.png[Sequence flow conditions]
+
+. Click the *Place order* user task and create an end event.
++
+image::cases/place-order-endevent.png[Create an end event]
+
+. Click the exclusive gateway and create a new user task and drag the new task to align it below the *Place order* task.
```

Comment:
Question: Is exclusive gateway a UI element? It is the only mention in this doc.  If it is a label or other UI element then it should be in bold.